package.json

```diff
@@ -93,6 +93,7 @@
     "@types/node": "^14.11.5",
     "@types/tape": "^4.13.0",
     "@types/tape-catch": "^1.0.0",
+    "eslint": "^6.8.0",
```

Comment:
Is it intentional or accidental change here?

Reply:
Ah, yes - intentional - because I didn't get the linting to work, but as @cgewecke pointed out on the call, this was likely due to some `package-lock.json` problem on my side. Will hopefully remember to remove on last commit before I do a final push 😀 (don't want to dig deeper into this eslint problem on my side right now).